src/datasets/summarizations/utils/commons.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { cacheSummaries, combineQuerySummariesFactories } from './commons';
+import { Summary } from '../types';
+
+describe('cacheSummaries', () => {
+  const summaries: Summary[] = ['s1', 's2', 's3'].map(createMockSummaryFromText);

```

Comment:
If the mock data is pretty simple, it's a lot easier to read if you just write it out. This way the reader doesn't have to go into the helper method and figure out what what it outputs.

A good compromise is to maybe have some simple mock data written out and use the helper method to generate more complicated ones.